### PR TITLE
Added distinguish between stateful and stateless events.

### PR DESCRIPTION
**Pull request by VaclavStupka**

### CODE REVIEW

File: GraphicsForAxisEvents/src/main.ts [hunk 1]
```diff
@@ -40,6 +40,21 @@ function onEventMessage(event: Event) {
...
+      event.lastTimeout = null;
+    }, event.duration);
+  } else {
+    event.co.setEnabled(state);
+  }
+}
+
 function getSettings() {
   try {
     const path = "./localdata/"; //process.env.PERSISTENT_DATA_PATH;
```
Comment (CamStreamer) [inline GraphicsForAxisEvents/src/main.ts:60]: Tady bys mel asi mit ten zakomentovany kod?

File: GraphicsForAxisEvents/src/main.ts [hunk 2]
```diff
@@ -40,6 +40,21 @@ function onEventMessage(event: Event) {
   }
 }
 
+function onStatefulEvent(event: Event, state: boolean) {
+  if (event.duration >= 1) {
+    event.co.setEnabled(true);
```
Comment (CamStreamer) [inline GraphicsForAxisEvents/src/main.ts:45]: Neni tady chyba v logice? Kdyz ti prijde state = false tak zapnes overlay?

**Review by CamStreamer — CHANGES_REQUESTED** [review]

File: GraphicsForAxisEvents/src/main.ts (hunk 1 above)
Comment (VaclavStupka) [reply GraphicsForAxisEvents/src/main.ts]: !

File: GraphicsForAxisEvents/src/main.ts (hunk 2 above)
Comment (VaclavStupka) [reply GraphicsForAxisEvents/src/main.ts]: !

**Review by CamStreamer — APPROVED** [review]

**Review by CamStreamer — APPROVED** [review]

**Pull request closed without merging**